### ISSUE TITLE
Explicitly set global label is filtered by filtrationMode

### ISSUE THREAD

**Issue by Mario-Hofstaetter** [mishamyte/serilog-sinks-grafana-loki#10]
Version `4.0.1` and `4.0.0`.
When using

```csharp
.WriteTo.GrafanaLoki("http://localhost:3100",
	filtrationLabels: new[] {"Service"}, // if omitted, Service is filtered too
	filtrationMode: LokiLabelFiltrationMode.Include,
	labels: new[] {new LokiLabel {Key = "Service", Value = Startup.Source.Name}},
```

one has to include label key `Service` in the `filtrationLabels` list or it will be dropped.
This is somewhat counterintuitive, because it is explicitly set as global label by the user.

It's perfectly fine to stay like that because the behavior is consistent,
but I'd like to suggest that global `lables` are not impacted by `filtrationMode`

BR

**Comment by mishamyte** [comment]
Yes, sounds reasonable. I have lost out of my scope, while redesigned filtration feature. Will fix

**Comment by mishamyte** [comment]
Fixed in [v4.0.2](https://github.com/mishamyte/serilog-sinks-grafana-loki/releases/tag/v4.0.2)

**Comment by Mario-Hofstaetter** [comment]
In `4.0.2` Global Labels are not impacted by `filtrationMode`, that goes for both `Include` and `Exclude` mode.
